gdal/frmts/ecw/ecwsdk_headers.h

```diff
@@ -31,6 +31,12 @@
 
 #include "cpl_port.h"
 
+#ifdef HAVE_CPL_SAFER_SNPRINTF
```

Comment:
all changes in that file seem undesirable. those additions were removed per d1a243f0171413dfb8e9336e7f6ecde43614d185